src/styles/hub/admin-bar.scss

```diff
@@ -18,18 +18,17 @@
 	}
 
 	/* the bell icon */
-	.ab-item {
+	.hub-icon {
```

Comment:
Changing this name to reflect it's purpose, and the case `ab-item` has been moved up higher in the DOM structure.